bpf/lookup.h

```diff
@@ -154,10 +154,69 @@ flow_stats_account(struct ebpf_headers_t *headers,
     return;
 }
 
+static inline void
+ovs_flow_mask_key(struct bpf_flow_key *dst, struct bpf_flow_key *src,
+                  struct bpf_flow_key *mask)
+{
+    const long *m = (const long *) mask;
+    const long *s = (const long *) src;
+    long *d = (long *) dst;
+    int i;
+
+    #pragma unroll
+    for (i = 0; i < sizeof *dst; i += sizeof(long)) {
+        *d++ = *s++ & *m++;
+    }
+}
+
+static inline struct bpf_action_batch *
+megaflow_lookup(struct bpf_flow_key *key OVS_UNUSED)
```

Comment:
maybe remove OVS_UNUSED if it not useful.

Reply:
Sure, I will remove that in the next version. Thanks